appcenter-post-clone.ps1

```diff
@@ -4,4 +4,26 @@
 # Licensed under the MIT License.
 
 echo "Executing post clone script in $pwd"
-(Get-Content NuGet.config).replace('NUGET_PASSWORD', $env:NUGET_PASSWORD) | Set-Content NuGet.config
+
+$nugetFileName = "NuGet.config"
+$contentValue = "<?xml version=""1.0"" encoding=""utf-8""?>`n" +
```

Comment:
Could probably be simplified using cat as shown in https://stackoverflow.com/a/23930212/598520

Reply:
It's PowerShell, not bash, so syntax will be different